packages/plugin-baby/src/EventedContainer.ts

```diff
@@ -0,0 +1,68 @@
+import { Container, interfaces } from "inversify";
```

Comment:
I can't find any test for this, would you write them while extracting?

Reply:
There are non :trollface:

Reply:
Okay, does it make sense to add them now?